src/support/utf8.c

```diff
@@ -615,71 +615,63 @@ size_t u8_printf(const char *fmt, ...)
     return cnt;
 }
 
-/* based on the valid_utf8 routine from the PCRE library by Philip Hazel
+/* Rewritten completely, original code not based on anything else
 
    length is in bytes, since without knowing whether the string is valid
    it's hard to know how many characters there are! */
-int u8_isvalid(const char *str, size_t length)
+int u8_isvalid(const char *iStr, size_t iLength)
```

Comment:
OK, I just copied this from my own UTF handling code, and changed the name to match Jeff's name, and removed the documentation :sad:
When I have all of u8 rewritten to run faster, can I use my standard conventions, since there will be no other surrounding code?